.github/workflows/CI.yml

```diff
@@ -56,16 +57,21 @@ jobs:
       - name: Test a full build of the app
         run: npm run build
 
-      - name: End-to-end tests on GitHub (Linux)
+      - name: End-to-end tests on Chrome (Linux)
```

Comment:
It's not required to run tests on each browser individually. If you think it is not good I can revert `CI.yml`

Reply:
I think it's fine to have more granularity. It does seem more structured like that, so keep this change.